src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    title,
+    user,
+    completed,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={
+        completed
+          ? 'TodoInfo TodoInfo--completed'
+          : 'TodoInfo'
+      }
```

Comment:
Better install [classnames](https://www.npmjs.com/package/classnames) and use it instead of ternary operator.
The ternary operator is very limited in its functionality to be used in classNames. (this way usually did in real projects and would be great to work with it now and would be needed to do in test tasks).